docs/Useful Info/Tracking-Request-Retrying.md

```diff
@@ -31,7 +34,8 @@ Temporary reasons can become permanent when the `status` changes to `failed`:
  * `duplicate` when the shipment already existed,  
  * `expired` when the tracking request was created more than 7 days ago and still not succeded,
  * `retries_exhausted` if we tried for 14 times to no avail,
- * `not_found` if the shipping line could not find the number.
+ * `not_found` if the shipping line could not find the BL number.
```

Comment:
@mattyturner 

`BL number` or just `the number`?

Reply:
I think it's `BL number` because if it's a booking number it will eventually transition to `retries_exhausted` ?

Reply:
Didn't you create a whimsical for this? :)